schema.graphql

```diff
@@ -662,117 +669,19 @@ type Disbursed @entity {
     timestamp: BigInt!
 }
 
-# Trader Historical Data
-
-type TraderDayData @entity {
-    "format: <trader id>-<day id>"
-    id: ID!
-    trader: Trader!
-    "timestamp"
-    date: BigInt!
-    "trading volume aggregated per day for trader"
-    tradingVolume: BigDecimal!
-    "trading fee aggregated per day for trader"
-    fee: BigInt!
-    "total pnl aggregated per day for trader"
-    realizedPnl: BigInt!
-}
-
 # Referral
 
-# For tracking day trade metrics for referral codes on a per trader basis
-type ReferralCodeTraderDayData @entity {
-    "format: <referal code>-<day id>-<trader>"
-    id: ID!
-    "relationship to trader. since 2022-08-14, for contract traders, this value is changed from the tx sender to contract. eoa traders have no change"
-    trader: Trader!
-    "relationship to referral day data"
-    referralCodeDayData: ReferralCodeDayData!
-    "trading volume in notional value"
-    tradingVolume: BigDecimal!
-    "fees in notional value"
-    fees: BigDecimal!
-}
-
-type ReferralCodeDayData @entity {
-    "format: <referal code>-<day id>"
-    id: ID!
-    referralCode: ReferralCode!
-    "timestamp"
-    date: BigInt!
-    "total trading volume in notional value"
-    tradingVolume: BigDecimal!
-    "total fees paid during day"
-    fees: BigDecimal!
-    "trading volume and fees grouped by trader"
-    traderData: [ReferralCodeTraderDayData!]! @derivedFrom(field: "referralCodeDayData")
-    "a list of new referees added today"
-    newReferees: [Trader!]!
-    "a list of referees which have traded on the day"
-    activeReferees: [Trader!]!
-}
-
 type ReferralCode @entity {
+    "format: referralCode"
     id: ID!
     "referrer/partner address"
     referrer: Trader!
     "list of registered referees"
     referees: [Trader!]! @derivedFrom(field: "refereeCode")
     "date created at"
     createdAt: BigInt!
-    "day data link"
-    dayData: [ReferralCodeDayData!] @derivedFrom(field: "referralCode")
-    "if user is vip tier"
-    vipTier: String
-    "how long user has been vip since"
-    vipSince: BigInt
```

Comment:
These 3 fields should be preserved I think. Although we don't use it in vePERP referral now, but just in case we want to know which partner has ever been VIP tier.